ros_ign_bridge/include/ros_ign_bridge/bridge.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Open Source Robotics Foundation, Inc.
+// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2018 Open Source Robotics Foundation, Inc.
```

We can leave the original copyright year, since we're porting existing code to ROS 2. This applies to other files too.